dqc_us_rules/util/facts.py

```diff
@@ -331,6 +331,33 @@ def legal_entity_axis_facts_by_member(facts):
     return results
 
 
+def axis_member_fact(axis_name, member_name, model_xbrl):
```

Comment:
can there only be one fact? seems like you're only returning the first found, but couldn't there potentially be many?